docs/changelog.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Place any unreleased changes here, that are subject to release in coming versions :).
 
+## 1.7.0 - 2025-06-10
```

Comment:
Should we add user relevant change-logs whence all the features have been fully migrated and tested?